### PR TITLE
introduce expander

### PR DESCRIPTION
**Pull request by drahnr**
Avoid clutter in the codebase, that just complicates things and isn't polkadot specific at all.

### CODE REVIEW

File: node/overseer/overseer-gen/README.md [hunk 1]
```diff
@@ -76,3 +76,12 @@ subsystems must be initialized only once (another compile time check) or be _rep
 a special setter like method `replace_<subsystem>`.
 
 A task spawner and subsystem context are required to be defined with `SpawnNamed` and respectively `SubsystemContext` implemented.
+
+## Debugging
+
+As always, debugging is notoriously annoying with bugged proc-macros.
+
+Therefor [`expander`](https://github.com/drahnr/expander) is employed to yield better
```
Comment (sandreim) [inline node/overseer/overseer-gen/README.md:84]:
```suggestion
Therefore [`expander`](https://github.com/drahnr/expander) is employed to yield better
```

**Review by sandreim — APPROVED** [review]
LGTM + 2 nits

File: node/overseer/overseer-gen/README.md (hunk 1 above)
Comment (vstakhov) [reply node/overseer/overseer-gen/README.md:84]: As far as I know, it should be `Therefore, ` even :)

**Review by vstakhov — APPROVED** [review]
LGTM

File: node/overseer/overseer-gen/Cargo.toml [hunk 2]
```diff
@@ -20,3 +20,7 @@ pin-project = "1.0"
 
 [dev-dependencies]
 trybuild = "1.0.53"
+
+[features]
+default = []
+expand = ["polkadot-overseer-gen-proc-macro/expand"]
```
Comment (drahnr) [inline node/overseer/overseer-gen/Cargo.toml:26]:
```suggestion
expand = ["polkadot-overseer-gen-proc-macro/expand"]

```

File: node/overseer/Cargo.toml [hunk 3]
```diff
@@ -30,3 +30,4 @@ test-helpers = { package = "polkadot-primitives-test-helpers", path = "../../pri
 
 [features]
 default = []
+expand = ["polkadot-overseer-gen/expand"]
```
Comment (drahnr) [inline node/overseer/Cargo.toml:33]:
```suggestion
expand = ["polkadot-overseer-gen/expand"]

```

**Comment by drahnr** [comment]
bot merge

File: node/overseer/overseer-gen/README.md (hunk 1 above)
Comment (sandreim) [reply node/overseer/overseer-gen/README.md]: Therefor is also valid, but archaic

**Pull request merged**